azerothcore/azerothcore-wotlk
# fix(DB/Spell): Savory Deviate Delight auras should not stack

Co-Authored-By: Wyrserth <wyrserth@users.noreply.github.com>





## CHANGES PROPOSED:
-  These auras should not stack
-  



## TESTS PERFORMED:
- works



## HOW TO TEST THE CHANGES:


1. .add 6657 50
2. Use the fish, see that when you change form the aura changes too. It does not put aura on aura


## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
